src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.test.tsx

```diff
@@ -67,4 +72,18 @@ describe('<TranscriptsListItemInfo /', () => {
   it('renders ViewInApp component', () => {
     expect(wrapper.find(ViewInApp)).toHaveLength(1);
   });
+
+  it('hides Download component by default', () => {
+    expect(wrapper.find(InstantDownloadTranscript)).toHaveLength(0);
+  });
+
+  it('shows Download component by default if expandDownload is true', () => {
+    wrapper = mount(
+      <MemoryRouter>
+        <TranscriptsListItemInfo {...props} expandDownload={true} />
+      </MemoryRouter>
+    );
+
+    expect(wrapper.find(InstantDownloadTranscript)).toHaveLength(1);
+  });
```

Comment:
It's a little surprising that the two neighbouring tests initialise the wrapper component differently. Maybe add a minor refactor to remove the creation of the wrapper from beforeEach into a helper function that accepts an object of partial props, mixes them together with default props, mounts the component and returns the wrapper? Such as [here](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/shared/components/view-in-app/ViewInApp.test.tsx#L33-L43)?

Reply:
That's done

Reply:
two things:

- doesn't it need the MemoryRouter anymore?
- why do you call it mount**Default**TranscriptListItem? It no longer has default set of props in the last test.

Reply:
> doesn't it need the MemoryRouter anymore?

The test fails if I remove it?

>why do you call it mountDefaultTranscriptListItem? It no longer has default set of props in the last test. And mountComponent/renderComponent would be even shorter than mountTranscriptListItem.

Changed it to be `renderComponent`.